### PR TITLE
Adds fetch of NFTs for Soul Signs page

### PR DESCRIPTION
**Pull request by angelagilhotra**
### Screenshare
https://share.vidyard.com/watch/U7DYqTATNxZFYXNDX85deE?

### CODE REVIEW

**Comment by angelagilhotra** [comment]
- [ ] change graph api endpoint 
- [ ] generate images on the page (instead of fetching)

File: packages/dapp/src/utils/constants/twitter.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  shareIntent: 'https://twitter.com/intent/tweet?text=',
+  shareContent: 'Checkout my collection! <COLLECTION_LINK>' // @todo!
```
Comment (andytudhope) [inline packages/dapp/src/utils/constants/twitter.js:3]: shareContent: 'Come and make eternally beautiful marks with me at https://sign.kernel.community so we can reimagine what significant money actually means.'

Make sure to include a shareImage, which is the NFT itself, as that is also important for Twitter purposes.

**Review by andytudhope — APPROVED** [review]
You continue to do entirely magical work, thank you so much :pray: :heart:

File: packages/dapp/src/utils/constants/twitter.js (hunk 1 above)
Comment (angelagilhotra) [reply packages/dapp/src/utils/constants/twitter.js:3]: that would come from the SEO no?

**Pull request merged**